slang.h

```diff
@@ -1709,7 +1717,20 @@ extern "C"
         size_t reproDataSize,
         ISlangFileSystem* replaceFileSystem,
         ISlangFileSystemExt** outFileSystem);
-    
+
+    /*! @see slang::ICompileRequest::overrideDiagnosticSeverity */
+    SLANG_API void spOverrideDiagnosticSeverity(
+        SlangCompileRequest* request,
+        SlangInt messageID,
+        SlangSeverity originalSeverity,
```

Comment:
I figured providing the original severity is a way for making sure that the call matches the user intentions. Like, to avoid accidentally disabling error messages. But then, there is a separate check for errors in the sink, so maybe I should remove this originalSeverity thing altogether. Removing it just from the public API doesn't make much sense.

Reply:
Removed.